cloudfront.tf

```diff
@@ -95,24 +95,15 @@ resource "aws_cloudfront_distribution" "default" {
   }
 
   default_cache_behavior {
-    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    allowed_methods  = var.default_cache_behavior_allowed_methods
     cached_methods   = ["GET", "HEAD"]
     target_origin_id = "default"
-    compress         = true
+    compress         = try(var.default_cache_behavior_compress, false)
 
-    forwarded_values {
-      query_string = true
-      headers      = var.cloudfront_forward_headers
```

Comment:
By removing this line, this option will not be used anymore, right? We still need this in some cases